source/blood/src/loadsave.cpp

```diff
@@ -91,7 +91,7 @@ void LoadSave::Write(void *pData, int nSize)
         ThrowError("File error #%d writing save file.", errno);
 }
 
-void LoadSave::LoadGame(char *pzFile)
+void LoadSave::LoadGame(char *pzFile, bool demoWasPlayed)
```

Comment:
You could move `gDemo.at1` and `gDemo.Close()` here from `void LoadGame(CGameMenuItemZEditBitmap *pItem, CGameMenuEvent *event)` instead